src/components/Book.js

```diff
@@ -39,7 +40,7 @@ const Book = ({ id, title, author }) => {
         <section className="new-section3">
           <p>CURRENT CHAPTER</p>
           <h3>Chapter 10</h3>
-          <button type="button">Update Progress</button>
+          <button className="update-button" type="button">Update Progress</button>
         </section>
       </section>
     </article>
```

Comment:
- [ ] Please once more check the annotated diagram below and fix the changes. The progress bar should be shift a bit to the left, the size of the update bar is a bit larger than expected as you can see below.

| Zeplin :+1:  | Yours |
| --- | --- |
| ![Bookstore CMS](https://github.com/mryeminthu/bookstore/assets/63087491/30100c59-38cf-4f25-8378-9f22028a343b) | ![Screenshot from 2023-09-29 07-18-14](https://github.com/mryeminthu/bookstore/assets/63087491/24e71ecb-73ba-4356-8436-b04dbd7f8de6) |